File: api/treinaweb-flask-api-gastos-pessoais/api/entidades/transacao.py
class Transacao():
    def __init__(self, titulo, descricao, valor, tipo, data, conta):
        self.__titulo = titulo
        self.__descricao = descricao
        self.__valor = valor
        self.__tipo = tipo
        self.__data = data
        self.__conta = conta

    @property
    def titulo(self):
        return self.__titulo

    @titulo.setter
    def titulo(self, titulo):
        self.__titulo = titulo

    @property
    def descricao(self):
        return self.__descricao

    @descricao.setter
    def descricao(self, descricao):
        self.__descricao = descricao

    @property
    def valor(self):
        return self.__valor

    @valor.setter
    def valor(self, valor):
        self.__valor = valor

    @property
    def tipo(self):
        return self.__tipo

    @tipo.setter
    def tipo(self, tipo):
        self.__tipo = tipo

    @property
    def data(self):
        return self.__data

    @data.setter
    def data(self, data):
        self.__data = data

    @property
    def conta(self):
        return self.__conta

    @conta.setter
    def conta(self, conta):
        self.__conta = conta

File: app/treinaweb-flutter-gerenciador-gastos-pessoais/lib/models/transacao.dart
class Transacao {
  int id, conta;
  String titulo, descricao, data, tipo;
  double valor;

  Transacao({this.id, this.tipo, this.conta, this.titulo, this.descricao,
  this.data, this.valor});

  Map<String, dynamic> toJson() {
    return {
      'titulo': titulo,
      'descricao': descricao,
      'valor': valor,
      'data': data,
      'tipo': tipo,
      'conta_id': conta
    };
  }

  Transacao.fromJson(Map map) {
    id = map["id"];
    titulo = map["titulo"];
    descricao = map["descricao"];
    valor = map["valor"];
    data = map["data"];
    tipo = map["tipo"];
    conta = map["conta_id"];
  }

  Map<String, dynamic> toMap() {
    return {
      'id': id,
      'tipo': tipo,
      'conta': conta,
      'titulo': titulo,
      'descricao': descricao,
      'data': data,
      'valor': valor,
    };
  }

  Transacao.fromMap(Map map){
    id = map["id"];
    tipo = map["tipo"];
    conta = map["conta"];
    titulo = map["titulo"];
    descricao = map["descricao"];
    data = map["data"];
    valor = map["valor"];
  }
}